compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -14710,7 +14758,8 @@ private STNode parseXMLAtomicNameIdentifier(STNode identifier) {
      */
     private STNode parseXMLStepExpression(STNode lhsExpr) {
```

Comment:
Shall we update the doc comment grammer to `expression xml-step-start xml-step-extend*`?